dashboard/test/controllers/user_school_infos_controller_test.rb

```diff
@@ -0,0 +1,32 @@
+require 'test_helper'
+
```

Comment:
@madelynkasula Should there be a test to check if user is authenticated?  I noticed in some other controller test we run before_action :authenticate user?  I am curious because I am not sure if the current test coverage I have accounts for this.  I also see that we use "Devise" gem as well.

Reply:
yes, it would be valuable to check:

1) signed out user can't update UserSchoolInfo
2) signed in user can only update UserSchoolInfo that belongs to them